extension/index.js

```diff
@@ -38,10 +42,19 @@ async function init() {
     addDependencies(createBox('Dev Dependencies'), devDependencies);
   }
 
-  if (dependencies.length && !pkg.private) {
-    const link = html`<a class="npmhub-anvaka btn btn-sm">Visualize full tree`;
-    link.href = `http://npm.anvaka.com/#/view/2d/${esc(pkg.name)}`;
-    dependenciesBox.insertBefore(link, dependenciesBox.firstChild);
+  if (!pkg.private) {
+    const realPkg = await window.backgroundFetch(getPkgUrl(pkg.name));
+    if (realPkg.name) { // if 404, realPkg === {}
+      const link = html`<a class="btn btn-sm">Open on npmjs.com`;
```

Comment:
EOF

The string is parsed on its own so when it ends, all open tags are also closed. Technically not valid HTML5 but it still works. This on the other hand is fully valid:

<img width="391" alt="valid HTML5" src="https://cloud.githubusercontent.com/assets/1402241/25937054/d5b1d3a6-365c-11e7-8f5c-c323758fbde0.png">